plutus-core/generators/Language/PlutusCore/Generators/NEAT/Spec.hs

```diff
@@ -90,16 +89,12 @@ tests genOpts@GenOptions{} =
       (Type ())
       (packAssertion prop_normalTypesCannotReduce)
 
-  -- note: we don't test type preservation of CEK as it does not
-  -- preserve type annotations although it would work for unit.
   , bigTest "type preservation - CK"
       genOpts {genDepth = 18}
       (TyBuiltinG TyUnitG)
       (packAssertion prop_typePreservation)
 
-  -- note: the typed CEK vs CK test would fail if values of this type
-  -- include type annotations. This is not the case for unit.
-  , bigTest "typed CEK vs CK, typed CEK vs untyped CEK produce the same output"
+  , bigTest "CK vs untyped CEK produce the same output"
```

Comment:
I think this still should say "typed CK"

Reply:
I see below that you've removed "typed" from the comments as it now unambiguous to say CK and CEK. But, I would still prefer to stress whether it is typed or untyped personally. But, it's not a big deal.

Reply:
I just looked at "typed CEK vs CK", parsed it as "(typed CEK) vs CK" and mindlessly changed test names having that naming convention in mind. I'll fix that.

Reply:
I fixed comments in [this](https://github.com/input-output-hk/plutus/pull/2744/commits/2ba8dc4f57e61a727bef1ebf5e5aed00a9f670ec) commit. Does it look fine now?

Reply:
yes -thanks!